src/OrbitGl/DisassemblyReport.cpp

```diff
@@ -5,6 +5,10 @@
 #include "DisassemblyReport.h"
 
 std::optional<uint32_t> DisassemblyReport::GetNumSamplesAtLine(size_t line) const {
+  // The given line number will be 1-indexed, but `Disassembler` works with 0-indexed line numbers.
+  // So we subtract 1 here:
```

Comment:
Nit: This line in the comment is pretty much exactly saying what the next code line does, so I'd remove it (the above line is great and exactly covers the "why").

Reply:
Removed. Thanks!